### PR TITLE
Fix the bug of blacked wallpaper on some Linux distros.

### PR DESCRIPTION
**Pull request by mmoneib**
This is a proposed fix for issue #297  . The wallpaper is shown as black seemingly due to an underlying element on top of the wallpaper itself. I didn't go through debugging how come this element exists and wasn't styled -- it doesn't look like there should be an element from the code in the first place -- and since the issue only affects certain distros, I thought a styling workaround would be a good approach.

In short, the styling workaround forces transparency on any descendant of a transparent folder if it doesn't have a styled background.

I tested on my machine, Fedora 32, and the fix looks fine.

### CODE REVIEW

**Comment by spheras** [comment]
Great @mmoneib! I have been checking in elementary, and it seems the changes have not any not desirable side effect.  I'm merging it.
Thak you very much!

**Comment by kerroncheddie** [comment]
Sorry for the late update, the change has solved the problem on debian SID, gnome-flashback.
Thank you for this amazing program.
